changelogs/fragments/javagateway.yml

```diff
@@ -0,0 +1,7 @@
+minor_changes:
+  - github workflow - Updated workflow files to checked against all supported combinations
+  - documentation - Updated role documentation to reflect changes
+breaking_changes:
```

Comment:
all changes here need to be prepended with role name otherwise resulting changelog will be a mess

Reply:
Updates Made